### PR TITLE
Small savings

### PR DESCRIPTION
**Pull request by comradekingu**
".NET" spotted by @rffontenelle :) As per https://en.wikipedia.org/wiki/.NET_Framework

### CODE REVIEW

File: weblate_web/templates/donate/form.html [hunk 1]
```diff
@@ -67,7 +67,7 @@ <h1 class="section-title min-m">{% trans "Your donation" %}</h1>
                 </form>
             </div>
             <p class="small">
-                {% trans "Please note—a full invoice for your donation will be issued immediatelly after payment." %}
+                {% trans "Please note—a full invoice for your donation is issued immediatelly after payment." %}
```
Comment (nijel) [inline weblate_web/templates/donate/form.html]: Shouldn't it be "after the payment"?

Comment (comradekingu) [reply weblate_web/templates/donate/form.html]: This is just the general, like "when payment occurs"
After watering, the lawn will be watered. The watering of the lawn is the action.

File: weblate_web/templates/snippets/pricing.html [hunk 2]
```diff
@@ -2,10 +2,10 @@
 {% load humanize %}
 	<section class="pricing">
     	<div class="row">
-            <h2 class="section-title">{% trans "<strong>Gratis</strong> or <strong>Premium</strong> — it’s up to you" %}</h2>
+            <h2 class="section-title">{% trans "<strong>Gratis</strong> or <strong>Premium</strong>—it’s up to you" %}</h2>
```
Comment (comradekingu) [inline weblate_web/templates/snippets/pricing.html]: I don't think selling price focuses on the service offered. (This is not the Wikipedia model)
How about "for your every need" or similar?

File: weblate_web/templates/download.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
         <div class="wrap">
             <h1 class="section-title min-m">{% trans "Download Weblate" %}</h1>
             <div class="page-desc">
-                <p>{% trans "Weblate comes with complete source code that you can use, edit or alter to meet your needs. Feel free to let your associates know about Weblate." %}</p>
+                <p>{% trans "All Weblate source code is freely available for you to use, modify, share and distribute. Discuss it with your associates." %}</p>
```
Comment (comradekingu) [inline weblate_web/templates/download.html]: Possibly "Let your associates know"?

Comment (nijel) [reply weblate_web/templates/download.html]: Hard to tell for me, this goes beyond by English knowledge...

**Pull request merged**